packages/salesforcedx-vscode-replay-debugger/src/states/frameEntryState.ts

```diff
@@ -19,11 +20,25 @@ export class FrameEntryState extends FrameState implements DebugLogState {
 
   public handle(logContext: LogContext): boolean {
     const sourceUri = logContext.getUriFromSignature(this._signature);
+    const frame = new ApexDebugStackFrameInfo(
+      logContext.getFrames().length,
+      this._signature
+    );
+    const id = logContext.getFrameHandler().create(frame);
+    const className =
+      this._signature.indexOf('.') > -1
+        ? this._signature.substring(0, this._signature.lastIndexOf('.'))
+        : this._signature;
+    if (logContext.getStaticVariablesClassMap().has(className)) {
```

Comment:
We get the className (and namespace if it has one) and checks to see if there was a previous frame with it. If so then we assign it the same static variables map. If not we create a new mapping.